viam-cartographer_test.go

```diff
@@ -34,6 +36,10 @@ var (
 	testMapRateSec = 200
 	_true          = true
 	_false         = false
+	timestamp1     = time.Time{}
```

Comment:
remove timestamp1/2 as they are always defined by the function. _zeroInt and _zeroTime can remain as they are valid comparators

Reply:
ok